openedx_blockstore_relay/management/commands/transfer_to_blockstore.py

```diff
@@ -0,0 +1,95 @@
+"""
+Transfers a Block (unit or sequence) and its components from the Open edX modulestore to Blockstore.
+"""
+from __future__ import absolute_import, print_function, unicode_literals
+
+import logging
+from argparse import ArgumentError
+from uuid import UUID
+
+from django.core.management.base import BaseCommand
+from future.moves.urllib.parse import urlparse
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import UsageKey
+
+from ...transfer_data import transfer_to_blockstore
+
+
+class Command(BaseCommand):
+    """
+    transfer_to_blockstore management command.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(Command, self).__init__(*args, **kwargs)
+        self.options = {}
+        self.help = __doc__
+        self.logger = logging.getLogger()
+        self.args = {}
+
+    def add_arguments(self, parser):
+        """
+        Add named arguments.
+        """
+        self.args['block'] = parser.add_argument(
+            '--block',
+            type=str,
+            required=True,
+            metavar='USAGE_KEY',
+            help='Usage key of the source Open edX Unit block, '
+                 'e.g., "block-v1:edX+DemoX+Demo_Course+type@html+block@030e35c4756a4ddc8d40b95fbbfff4d4"'
+        )
+        self.args['bundle'] = parser.add_argument(
+            '--bundle',
+            type=str,
+            required=True,
```

Comment:
Or is it better to take a `collection_uuid`, and create a new Bundle for each child of the given `block`?

For example, if you specify a collection for each of these block types, you would get:
* block `type@course` --> Section (`chapter`) in each Bundle
* block `type@chapter` --> Subsection (`sequential`) in each Bundle
* block `type@sequential` --> Unit (`vertical`) in each Bundle.
* block `type@vertical` --> Subunit (`html`, `problem`, `video`) in each Bundle.

Reply:
I would recommend keeping it simple for now and just putting everything in one Bundle. This will have to be refactored for the data transfer automation later anyway once there is more clarity about the product requirements.

Reply:
Ok cool, thanks @symbolist . I'll make the command accept *either* a `--bundle--uuid` (like it does currently) or a `--collection--uuid` (which will auto-create a target bundle), and import everything into that single bundle.